strapi/strapi
# feat(review-workflow): Workflow content type and controllers

### What does it do?

Add in a Workflow content-type and associated controllers
The first brick of the review-workflow feature. 

### How to test it?

1. Manually create a workflow in Strapi with `admin::workflow` create query
2. Use newly created routes to get it
  a. GET `/admin/review-workflows/workflows` that should give you a list
  b. GET `/admin/review-workflows/workflows/:workflow_id` that should give you the selected workflow
